applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -297,29 +297,29 @@ def Initialize(self):
     def InitializeSolutionStep(self):
         # Run the primal analysis.
         # TODO if primal_analysis.status==solved: return
-        Logger.PrintInfo("\n> Starting primal analysis for response:", self.identifier)
+        Logger.PrintInfo("AdjointResponseFunction", "Starting primal analysis for response:", self.identifier)
```

Comment:
Is there maybe a simple possibility to make this label variable so that it reflects the actually used adjoint response function (e.g. something like AdjointMaxStressResponse)?

Reply:
What do you prefer?

```
AdjointResponseFunction::MaxStress: Automatic set up adjoint parameters for response: max_stress 
AdjointResponseFunction::MaxStress: Output process is removed for adjoint analysis. To enable it define adjoint_parameters yourself. 
```

```
AdjointMaxStressResponseFunction: Automatic set up adjoint parameters for response: max_stress 
AdjointMaxStressResponseFunction: Output process is removed for adjoint analysis. To enable it define adjoint_parameters yourself. 
```

Reply:
I would go for the lower one. What do you think about removing the term "Function". I mean its clear that it is a function. So AdjointMaxStressResponse is maybe be descriptive enough. Your call.

Reply:
```
AdjointMaxStressResponse: Automatic set up adjoint parameters for response: max_stress 
AdjointMaxStressResponse: Output process is removed for adjoint analysis. To enable it define adjoint_parameters yourself.
```